src/components/editor/ResourceTemplateForm.jsx

```diff
@@ -72,36 +72,19 @@ class ResourceTemplateForm extends Component {
     }
   }
 
-  getContentForModalButton = (rtId) => {
-    let content
-    let resourceTemplate = getResourceTemplate(rtId)
-    const pt = resourceTemplate.propertyTemplates[0]
-    if (this.isLiteralWithDefaultValue(pt)) {
-      content = pt.valueConstraint.defaults[0].defaultLiteral
-    }
-    return content
-  }
-
   // Note: rtIds is expected to be an array of length at least one
   resourceTemplateFields = (rtIds) => {
     const rtProperties = []
-    rtIds.map((rtId, i) => {
-      rtProperties.push(<PropertyResourceTemplate key={shortid.generate()} resourceTemplate={getResourceTemplate(rtId)} />)
+    rtIds.map(async (rtId, i) => {
+      let template = await getResourceTemplate(rtId)
+      rtProperties.push(<PropertyResourceTemplate key={shortid.generate()} resourceTemplate={template} />)
       if ((rtIds.length - i) > 1) {
         rtProperties.push(<hr key={i} />)
       }
     })
     return rtProperties
   }
 
-  defaultValues = () => {
```

Comment:
This duplicates what's on line 31 above. Cruft.

Reply:
Fixed in PR #453